examples/tutorials/advance_model_config/02_device.ipynb

```diff
@@ -0,0 +1,490 @@
+{
```

Comment:
<p>add a sentence about what's happening&nbsp;here and below</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>